### DISCUSSION TITLE
Train Panoptic model

### DISCUSSION THREAD

**Discussion by SmadarCohen111** [facebookresearch/detectron2#3858] [Q&A]
Hi, I'm trying to train panoptic model on my own dataset, but I don't understand exactly how the data format looks like, when I register the data its seems like everting works fine, but when I try to train, I get the following warnings:

[01/06 11:46:12 d2.data.datasets.coco]: Loaded 1704 images in COCO format from (....)
WARNING [01/06 11:46:12 d2.data.datasets.coco]: Directory (...) and (...) has 0 and 1704 files, respectively.
WARNING [01/06 11:46:12 d2.data.datasets.coco]: Will use their intersection of 0 files.
[01/06 11:46:12 d2.data.datasets.coco]: Loaded 0 images with semantic segmentation from (..)

and the following error:

assert len(sem_seg_file_to_entry) > 0
AssertionError

I used the cocopanopticapi to convert the data, can someone plz help me figure out where I wrong?

**Comment by zhanghang1989** [comment]
Looks like the dataset doesn't have both thing and stuff annotations.
